app/code/Magento/Wishlist/view/frontend/web/css/styles.less

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Please remove this file and move styles to both `app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less` and `app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less`

Reply:
ok